cmake/modules/configuration_files.cmake

```diff
@@ -78,9 +78,15 @@ else()
   message(FATAL_ERROR "No prj.conf file was found in the ${APPLICATION_CONFIG_DIR} folder, "
                       "please read the Zephyr documentation on application development.")
 endif()
+zephyr_file_suffix(CONF_FILE)
```

Comment:
This doesn't work consistently when I pass `-DCONF_FILE=<relative-path>`, as it depends on CMake's working directory. The same goes for other variables.

Sidenote: I caught this because sysbuild changes the working dir for each image, which in itself is confusing.

Reply:
Fixed